2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -134,39 +134,41 @@ function replset_addr() {
 }
 
 # mongo_remove removes the current MongoDB from the cluster
+# optionally takes address of replset to connect and address of server to remove
 function mongo_remove() {
   local host
   # if we cannot determine the IP address of the primary, exit without an error
   # to allow callers to proceed with their logic
-  host="$(replset_addr || true)"
+  host="${1:-$(replset_addr || true)}"
+  local mongo_addr
+  mongo_addr="${2:-(mongo_addr)}"
+
   if [ -z "$host" ]; then
     return
   fi
 
-  local mongo_addr
-  mongo_addr="$(mongo_addr)"
-
   echo "=> Removing ${mongo_addr} from replica set ..."
   mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" \
-    --host "${host}" --eval "rs.remove('${mongo_addr}');" || true
+    --host "${host}" --quiet --eval "rs.remove('${mongo_addr}');" || true
 }
 
 # mongo_add advertise the current container to other mongo replicas
+# optionally takes address of replset to connect and address of server to add
 function mongo_add() {
   local host
   # if we cannot determine the IP address of the primary, exit without an error
   # to allow callers to proceed with their logic
-  host="$(replset_addr || true)"
+  host="${1:-$(replset_addr || true)}"
+  local mongo_addr
+  mongo_addr="${2:-$(mongo_addr)}"
+
   if [ -z "$host" ]; then
     return
   fi
 
-  local mongo_addr
-  mongo_addr="$(mongo_addr)"
-
   echo "=> Adding ${mongo_addr} to replica set ..."
   mongo admin -u admin -p "${MONGODB_ADMIN_PASSWORD}" \
-    --host "${host}" --eval "rs.add('${mongo_addr}');"
+    --host "${host}" --quiet --eval "rs.add('${mongo_addr}');"
```

Comment:
Can you add `printjson()` here? Like this: `--eval "rs.add(printjson('${mongo_addr}'));"` Because otherwise I sill won't be able to use this function in my PR.